javascript/script.js

```diff
@@ -187,6 +187,44 @@ mainForm.addEventListener('submit', (event) => {
   event.preventDefault();
 
   if (emailValid === true) {
+    localStorage.removeItem('fullNameValue');
+    localStorage.removeItem('emailValue');
+    localStorage.removeItem('textareaValue');
     mainForm.submit();
   }
 });
+
+/*
+Local Storage
+*/
+const fullNameInput = document.getElementById('full-name');
+const emailInput = document.getElementById('email-form');
+const textareaInput = document.getElementById('contact-form-text-area');
+
+fullNameInput.addEventListener('change', () => {
+  localStorage.setItem('fullNameValue', fullNameInput.value);
+});
+
+emailInput.addEventListener('change', () => {
+  localStorage.setItem('emailValue', emailInput.value);
+});
+
+textareaInput.addEventListener('change', () => {
+  localStorage.setItem('textareaValue', textareaInput.value);
+});
+
+const fullNameValue = localStorage.getItem('fullNameValue');
+const emailValue = localStorage.getItem('emailValue');
+const textareaValue = localStorage.getItem('textareaValue');
```

Comment:
- [ ] Kindly be noticed that you should add all the values of the form in a single `localstorage` variable and you should use `JSON.parse` and `JSON.stringify`.
- [ ] example like this:-
- [ ] `localStorage.setItem('contactFormInput', JSON.stringify(formData));`
- [ ] `let savedFormData = localStorage.getItem('contactFormInput');`
`savedFormData = JSON.parse(savedFormData);`